lib/manager/terraform/util.ts

```diff
@@ -42,3 +45,32 @@ export function checkIfStringIsPath(path: string): boolean {
   const match = pathStringRegex.exec(path);
   return !!match;
 }
+
+export function massageProviderLookupName(dep: PackageDependency): void {
```

Comment:
Do we really need this?

Reply:
I use it to encapsulate the massaging code and I reuse it now in the `updateArtifacts` function